dask/diagnostics/tests/test_profiler.py

```diff
@@ -270,6 +277,8 @@ def test_resource_profiler_plot():
     rprof.clear()
     for results in [[], [(1.0, 0, 0)]]:
         rprof.results = results
+        rprof.start_time = 0.0
+        rprof.end_time = 1.0
```

Comment:
Why is this needed?

Reply:
Forgive me if this is incorrect, it's been a while since I looked at this. But from what I see in this PR, the `rprof.clear()` call before this `for` loop sets `start_time` and `end_time` to `None` (which matches what was in the clear method before this PR). Now that `start_time` and `end_time` are actually used for something in the visualization they can't be `None` because of the arithmetic operations being done with them.

...yeah, this line (218 of profile.py):

https://github.com/dask/dask/pull/9739/files#diff-d1ec632f95ddf2554df9d54f70b8980b9f24699be167aa97c8b724c18a7da88dR218

Previously only `results` were sent to the visualize function for the ResourceProfiler.

Reply:
end_time here is arbitrary picked just to visualize the for the test, correct ?  It could have also been a call to the current time?

Reply:
Yes. I think that's right.

Reply:
Looking it over more, this start and end setting could be seen as the initialization of fake results/usage of the profiler. I suppose this could be made more clear with a fixture or a helper context manager.